setup.py

```diff
@@ -10,11 +10,11 @@
 
 setuptools.setup(
     name="nionswift-experimental",
-    version="0.4.0",
+    version="0.5.0",
     author="Nion Software",
     author_email="swift@nion.com",
     description="Experimental tools package for Nion Swift.",
-    packages=["nionswift_plugin.nion_experimental_tools"],
+    packages=["nionswift_plugin.eels_tools", "nionswift_plugin.filters", "nionswift_plugin.misc"],
```

Comment:
These package names are too general. "filters" and "eels_tools" are essentially top level names. How about leaving this package the same name as before, adding your code into init.py and adding the UI file. then adding the experimental library at the level above this. Or adding your 'nion_experimental_4dtools' under 'nionswift_plugin'?